components/tfx/Trainer/component.py

```diff
@@ -5,18 +5,19 @@
 
 def Trainer(
     examples_path: InputPath('Examples'),
-    transform_output_path: InputPath('TransformGraph'), # ? = None
-    #transform_graph_path: InputPath('TransformGraph'),
     schema_path: InputPath('Schema'),
 
-    output_path: OutputPath('Model'),
+    model_path: OutputPath('Model'),
 
+    train_args: {'JsonObject': {'data_type': 'proto:tfx.components.trainer.TrainArgs'}},
+    eval_args: {'JsonObject': {'data_type': 'proto:tfx.components.trainer.EvalArgs'}},
     module_file: str = None,
     trainer_fn: str = None,
```

Comment:
will you have another CL to address the changes for native Keras (which will replace Estimator?)

Or name it as "run_fn" in this PR? May prefer you split it to another PR.

Reply:
These components just call the existing TFX components. When TFX components switch to Keras, these components will also switch.